.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+Thanks for raising a Spring Integration issue. What sort of issue are you raising?
+
+Question
+
+Please ask questions about how to use something, or to understand why something isn't
+working as you expect it to, on Stack Overflow using the `spring-integration` tag.
```

Comment:
How about a wrapping "Stack Overflow" to link?
Looks like not everybody knows what is that.